third_party/openmetadata/pvc1.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: dag
+  namespace: INSTALLATION_NAMESPACE
```

Comment:
Do not specify the namespace

Reply:
removed namespace wherever it is unneeded.
It is still needed for `PersistentVolume` definitions